.travis.yml

```diff
@@ -2,9 +2,11 @@ before_script:
  - git fetch --unshallow # we need a full clone for tests
  - git config --global user.email "you@example.com"
  - git config --global user.name "Your Name"
+ - gem install bundler
```

Comment:
I have solved the problem.
(Sorry, I made a mistake `before_install` for `before_script`.)